numpy/polynomial/chebyshev.py

```diff
@@ -104,7 +104,7 @@
 ----------
 .. [1] A. T. Benjamin, et al., "Combinatorial Trigonometry with Chebyshev
   Polynomials," *Journal of Statistical Planning and Inference 14*, 2008
-  (preprint: https://www.math.hmc.edu/~benjamin/papers/CombTrig.pdf, pg. 4)
+  (https://web.archive.org/web/20080221202153/https://www.math.hmc.edu/~benjamin/papers/CombTrig.pdf, pg. 4)
```

Comment:
There seems to be a non-preprint, non-web.archive link to this paper available from the original institution: https://math.hmc.edu/benjamin/wp-content/uploads/sites/5/2019/06/Combinatorially-Composing-Chebyshev-Polynomials.pdf

Reply:
Sounds good - my only other thought is that the original link was a preprint, whereas the newer link is post publication (and peer review). I don't have a strong preference either way.